mods/content/polaris/datum/locations.dm

```diff
@@ -6,24 +170,347 @@
 	Though fiercely independent for a few centuries, \
 	today, Sif is united with Kara under the Vir Governmental Authority as \
 	a full SolGov member state, and acts as head of the Golden Crescent Alliance Bloc."
-	ruling_body = "Vir Governmental Authority"
+	capital = "New Reykjavik"
+	local_government = "Vir Governmental Authority"
+	ruling_body = "Golden Crescent Alliance Bloc"
+	distance = "67.4 ly"
 
 /decl/background_detail/location/kara
 	name = "Kara, Vir"
 	uid = "location_kara"
 	description = "A series of asteroid habitats orbitting the gas giant Kara in the Vir system.\
 	Originally populated by smugglers, pirates, and their descendants, \
 	an internal police action and the discovery of phoron opened the region to exploitation \
-	by corporations like NanoTrasen, whose mining and colony stations now dominate the ring."
-	ruling_body = "Vir Governmental Authority"
+	by corporations like NanoTrasen, whose mining and colony stations now dominate the ring. \
+	Kara's habitats, once independent, are now under the aegis of the Vir Governmental Authority -\
+	but still retain a significant degree of autonomy."
+	capital = "New Reykjavik, Sif"
+	local_government = "Vir Governmental Authority"
+	ruling_body = "Golden Crescent Regional Government"
```

Comment:
These should all probably be "The Golden Crescent Alliance Bloc" or "Golden Crescent Regional Government" for consistency